xml/issue1203.xml

```diff
@@ -134,6 +143,58 @@ and only if <tt>Ostream</tt> is not an lvalue reference type and is derived from
 </ins></p>
 </blockquote>
 </blockquote>
+</blockquote>
+</discussion>
+
+<resolution>
+<p>This resolution is relative to N4750.</p>
+<p>
+Change <sref ref="[istream.rvalue]"/> as follows:
+</p>
+
+<blockquote><pre>
+template &lt;class <del>charT, class traits</del> <ins>Istream</ins>, class T&gt;
+  <del>basic_istream&lt;charT, traits&gt;&amp;</del> <ins>Istream&amp;&amp;</ins>
+  operator&gt;&gt;(<del>basic_istream&lt;charT, traits&gt;</del> <ins>Istream</ins>&amp;&amp; is, T&amp;&amp; x);
+</pre>
+<blockquote>
+<p>
+-1- <i>Effects:</i> Equivalent to:
+</p>
+<pre>
+<tt>is &gt;&gt; std::forward&lt;T&gt;(x)</tt>
```

Comment:
I don't think we need a `<tt>` here. `<pre>` should do it.

Also missing semicolon.